src/components/Text/Text.module.css

```diff
@@ -5,8 +5,78 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) h1, h2, h3, h4, h5, h6
+ * 2) Styles are either controlled by as or size declaration
  */
-/* stylelint-disable-next-line block-no-empty */
-.text {
+.text,
+.text--body, 
+.text--md {
+  @mixin reset;
```

Comment:
will this make it hard for people to set their own margins through tailwind? we could just set `margin-bottom` to match the existing component

Reply:
Will do